examples/step-71/doc/intro.dox

```diff
@@ -0,0 +1,678 @@
+    \boldsymbol{\mathbb{H}}}
+      {\left(h_{v}^{\text{sat}}\right)^{2}} \right)
+@f]
+and the evolution law
+@f[
+  \dot{\mathbf{C}}_{v} \left( \mathbf{C} \right)
+= \frac{1}{\tau} \left[
+      \left[\left[\text{det}\left(\mathbf{F}\right)\right]^{-\frac{2}{d}}
+        \mathbf{C}\right]^{-1}
+    - \mathbf{C}_{v} \right]
+@f]
+for the internal viscous variable.
+We've chosen the magnetoelastic part of the energy
+$\psi_{0}^{ME} \left( \mathbf{C}, \boldsymbol{\mathbb{H}} \right)$
+to match that of the first material model that we explored, so this part
+needs no further explanation. As for the viscous part $\psi_{0}^{MVE}$,
+this component of the free energy (in conjunction with the evolution law for
+the viscous deformation tensor) is taken from @cite Linder2011a (with the
+additional scaling by the viscous saturation function described in
+@cite Pelteret2018a). It is derived in a thermodynamically consistent
+framework that, at its core, models the movement of polymer chains on a
+micro-scale level.
+
+To proceed beyond this point, we'll also need to consider the time
+discretization of the evolution law.
+Choosing the implicit first-order backwards difference scheme, then
+@f[
+  \dot{\mathbf{C}}_{v}
+\approx \frac{\mathbf{C}_{v}^{(t)} - \mathbf{C}_{v}^{(t-1)}}{\Delta t}
+= \frac{1}{\tau} \left[
+      \left[\left[\text{det}\left(\mathbf{F}\right)\right]^{-\frac{2}{d}}
+        \mathbf{C}\right]^{-1}
+    - \mathbf{C}_{v}^{(t)} \right]
+@f]
+where the superscript $(t)$ denotes that the quantity is taken at the
+current timestep, and $(t-1)$ denotes quantities taken at the previous
+timestep (i.e., a history variable). The timestep size $\Delta t$ is the
+difference between the current time and that of the previous timestep.
+Rearranging the terms so that all internal variable quantities at the
+current time are on the left hand side of the equation, we get
+@f[
+\mathbf{C}_{v}^{(t)}
+= \frac{1}{1 + \frac{\Delta t}{\tau_{v}}} \left[
+    \mathbf{C}_{v}^{(t-1)}
+  + \frac{\Delta t}{\tau_{v}}
+    \left[\left[\text{det}\left(\mathbf{F}\right)\right]^{-\frac{2}{d}}
+    \mathbf{C} \right]^{-1}
+  \right]
+@f]
+that matches @cite Linder2011a equation 54.
+
+<h3>Rheological experiment</h3>
+
+Now that we have shown all of these formulas for the thermodynamics and theory
+governing magneto-mechanics and constitutive models, let us outline what the
+program will do with all of this.
+We wish to do something *meaningful* with the materials laws that we've formulated,
+and so it makes sense to subject them to some mechanical and magnetic loading
+conditions that are, in some way, representative of some conditions that might
+be found either in an application or in a laboratory setting. One way to achieve
+that aim would be to embed these constitutive laws in a finite element model to
+simulate a device. In this instance, though, we'll keep things simple (we are
+focussing on the automatic and symbolic differentiation concepts, after all)
```

Comment:
Apparently `focusing` is the right murican way to spell this. Cause we are not paying for extra letters over here.